### DISCUSSION TITLE
Ndc network protect allow 10119 exited with 127

### DISCUSSION THREAD

**Discussion by KadamovEugene** [Mygod/VPNHotspot#587] [Q&A] [answered]
Setup: Oneplus 3, Android 9, rooted with Magisk, the app is installed under /system/priv-app.
Issue: after Wi-Fi hotspot is enabled it is not possible to turn wlan0 connection, the error message as in the title appears (screenshots are attached, logs are also attached).
Any ideas what could resolve the problem, could it be android version or phone model issue?
![Screenshot_20240708-111900](https://github.com/Mygod/VPNHotspot/assets/60624920/38d3208c-231f-4a5f-9964-2e58cae49d49)
![Screenshot_20240708-111847](https://github.com/Mygod/VPNHotspot/assets/60624920/f0da40cc-f955-4118-b1e8-d97ec0748a9b)
[vpnhotspot-5100415682696595540.log](https://github.com/user-attachments/files/16125678/vpnhotspot-5100415682696595540.log)

**Comment by Mygod** [comment, marked as answer]
From your logs, it looks like your root is not working properly. Multiple commands cannot be located by `sh`. Maybe the `PATH` variable is not set properly?

```
ndc: not found
iptables: not found
logcat: not found
```

**Reply by KadamovEugene** [reply]
Yea, it looks like the root issue, thanks! I set up the app within OnePlus 9 pro and everything works just perfect.
Great application
